lnd_test.go

```diff
@@ -1098,7 +1094,7 @@ func testUpdateChannelPolicy(net *lntest.NetworkHarness, t *harnessTest) {
 	pushAmt := chanAmt / 2
 
 	// Create a channel Alice->Bob.
-	ctxt, _ := context.WithTimeout(ctxb, timeout)
+	ctxt, _ := context.WithTimeout(ctxb, channelOpenTimeout)
```

Comment:
There are some `subscribeGraphNotifications` calls above that are missing a timeout.

Reply:
We cannot give them a timeout, since they will need the context open for the entirety of the test.